gravitee-apim-console-webui/src/management/settings/api-quality-rules-ng/api-quality-rules-ng.component.ts

```diff
@@ -0,0 +1,262 @@
+import { GIO_DIALOG_WIDTH, GioConfirmDialogComponent, GioConfirmDialogData } from '@gravitee/ui-particles-angular';
+import { MatDialog } from '@angular/material/dialog';
+
+import {
+  ApiQualityRulesDialogData,
+  ApiQualityRulesDialogResult,
+  ApiQualityRulesNgAddDialogComponent,
+} from './api-quality-rules-ng-add-dialog/api-quality-rules-ng-add-dialog.component';
+
+import { Constants, EnvSettings } from '../../../entities/Constants';
+import { PortalSettingsService } from '../../../services-ngx/portal-settings.service';
+import { SnackBarService } from '../../../services-ngx/snack-bar.service';
+import { EnvironmentSettingsService } from '../../../services-ngx/environment-settings.service';
+import { QualityRuleService } from '../../../services-ngx/quality-rule.service';
+import { QualityRule } from '../../../entities/qualityRule';
+import { GioTableWrapperFilters } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { gioTableFilterCollection } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.util';
+
+interface ApiQualityRulesForm {
+  apiReview: FormGroup<{
+    enabled: FormControl<boolean>;
+  }>;
+  apiQualityMetrics: FormGroup<{
+    enabled: FormControl<boolean>;
+    descriptionWeight: FormControl<number>;
+    descriptionMinLength: FormControl<number>;
+    logoWeight: FormControl<number>;
+    categoriesWeight: FormControl<number>;
+    labelsWeight: FormControl<number>;
+    functionalDocumentationWeight: FormControl<number>;
+    technicalDocumentationWeight: FormControl<number>;
+    healthcheckWeight: FormControl<number>;
+  }>;
+}
+
+@Component({
+  selector: 'api-quality-rules-ng',
+  templateUrl: './api-quality-rules-ng.component.html',
+  styleUrls: ['./api-quality-rules-ng.component.scss'],
+})
+export class ApiQualityRulesNgComponent implements OnInit {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+  settings: EnvSettings;
+  apiQualityRulesForm: UntypedFormGroup;
+  public formInitialValues: unknown;
+  public qualityRulesListTable: QualityRule[] = [];
+  public displayedColumns = ['quality', 'description', 'weight', 'actions'];
+  public qualityRulesUnpaginatedLength = 0;
+  public filteredQualityRulesSettingsTable: QualityRule[] = [];
+  public isLoadingData = true;
+  public initialFilters: GioTableWrapperFilters = {
+    pagination: {
+      size: 10,
+      index: 1,
+    },
+    searchTerm: '',
+  };
+
+  constructor(
+    @Inject('Constants') private readonly constants: Constants,
+    private readonly portalSettingsService: PortalSettingsService,
+    private readonly snackBarService: SnackBarService,
+    private readonly environmentSettingsService: EnvironmentSettingsService,
+    private readonly qualityRuleService: QualityRuleService,
+    private readonly matDialog: MatDialog,
+  ) {}
+
+  public ngOnInit() {
+    this.isLoadingData = true;
+    this.settings = this.constants.env.settings;
```

Comment:
I'm wondering whether it's necessary to update the `this.constants.env.settings`

Reply:
It's probably a more global reflection. it's actually a guard that initiates it. but maybe we need to move it to the service and add a refresh subject. 
it's another subject to do in dedicated PR if needed

Reply:
I'll implement it in second PR